examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "/examples/src/main/resources/people.json")
```

Comment:
> This makes a very strong assumption about the location of the files right?

I thought so, but it results in failure because client try to access HDFS. The stack trace is here:
https://issues.apache.org/jira/browse/SPARK-18399?focusedCommentId=15653722&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15653722